### DISCUSSION TITLE
DetailsList - Column formatting for dates, numbers, amounts, etc...

### DISCUSSION THREAD

**Discussion by e-gallis** [microsoft/powercat-creator-kit#333] [Q&A]
Hi @mikefactorial, @BMeganPP, @denise-msft,

As a couple of questions here mention it, many makers would like to be able to format their **DetailsList** column values as they wish. I, myself, need to display a date information without the time next to it.
Using the **AddColumns**() function is just a temporary workaround as it is not delegable.

Would it be possible for you guys to add a "Format" property to this component so we can use it to specify format like the ones in the **Text**() Power Fx function? Like "_yyyy-mm-dd_" for a date-only display...

Thanks

**Comment by scottdurow** [comment]
I agree - the DetailsList does need a format string property for date columns (and numeric) - in the mean time you can use the delegation approach I show here https://youtu.be/zfyoBLbPvoQ (around the 19:00) mark
